doc/source/background.md

```diff
@@ -2,29 +2,97 @@
+Below, some more information on the people behind the package, its funding sources, and its mission.
 
 ## The people behind xDEM
 
+### Creation
+
 ```{margin}
-<sup>2</sup>More on our GlacioHack founder at [adehecq.github.io](https://adehecq.github.io/)!
+<sup>1</sup>More on our GlacioHack founder at [adehecq.github.io](https://adehecq.github.io/).
 ```
 
 xDEM was created during the **[GlacioHack](https://github.com/GlacioHack) hackathon**, that was initiated by
-Amaury Dehecq<sup>2</sup> and took place online on November 8, 2020.
+Amaury Dehecq<sup>1</sup> and took place online on November 8, 2020.
 
 ```{margin}
-<sup>3</sup>Check-out [glaciology.ch](https://glaciology.ch) on our founding group of VAW glaciology!
+<sup>2</sup>Check-out [glaciology.ch](https://glaciology.ch) on our founding group of VAW glaciology!
 ```
 
 The initial core development of xDEM was performed by members of the Glaciology group of the Laboratory of Hydraulics, Hydrology and
-Glaciology (VAW) at ETH Zürich<sup>3</sup>, with contributions by members of the University of Oslo, the University of Washington, and University
+Glaciology (VAW) at ETH Zürich<sup>2</sup>, with contributions by members of the University of Oslo, the University of Washington, and University
 Grenoble Alpes.
 
-We are not software developers but geoscientists, and we try our best to offer tools that can be useful to a larger group,
-documented, reliable and maintained. All development and maintenance is made on a voluntary basis and we welcome
-any new contributors! See some information on how to contribute in the dedicated page of our
-[GitHub repository](https://github.com/GlacioHack/xdem/blob/main/CONTRIBUTING.md).
+### Current team
+
+```{margin}
+<sup>3</sup>More on CNES's 3D missions on the [CO3D constellation page](https://cnes.fr/en/projects/co3d).
+```
+
+The current lead development team includes **researchers in Earth observation and engineers from
+[CNES](https://cnes.fr/en)** (French Space Agency). We specialize in elevation data analysis, for application in Earth
+science or for operational use for 3D satellite missions<sup>3</sup>.
+
+Other volunteer contributors span diverse scientific backgrounds in industry or research. We welcome
+any new contributors! See how to contribute on [the dedicated page of our repository](https://github.com/GlacioHack/xdem/blob/main/CONTRIBUTING.md).
+
+## Funding acknowledgments
+
+Members of the lead development team acknowledge funding from:
+- SNSF grant no. 184634, a MeteoSwiss [GCOS](https://gcos.wmo.int/en/home) project on elevation data analysis for glaciology,
+- NASA award 80NSSC22K1094, an [STV](https://science.nasa.gov/earth-science/decadal-surveys/decadal-stv/) project on the fusion of elevation data,
+- NASA award 80NSSC23K0192, an [ICESat-2](https://icesat-2.gsfc.nasa.gov/) project on the processing of elevation data in the cloud,
+- CNES (French Space Agency): ?.
```

Comment:
do you need a number ? I don't know which award would be "public" 
here. Maybe: study on merging demcompare and xdem and development of demcompare functions into xdem
or better written ;)

Reply:
No number necessary, I added the sentence! :)